nautobot/extras/filters/__init__.py

```diff
@@ -922,11 +923,16 @@ class JobResultFilterSet(BaseFilterSet, CustomFieldModelFilterSetMixin):
         queryset=Job.objects.all(),
         label="Job (ID) - Deprecated (use job_model filter)",
     )
+    scheduled_job = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
Unless I'm missing it, the `JobResultTable` doesn't appear to include a `scheduled_job` column as an option. Might be good to add that too?

Reply:
You're correct, it's not in the table.  I can add it in though.